### DISCUSSION TITLE
Still unable to verify my status as an online MS student with Georgia Tech.

### DISCUSSION THREAD

**Discussion by metamutator** [community/community#65337] [GitHub Education]
### Hi everyone,

I am an online MS student studying at Georgia Tech.

I have:
1. Associated my GitHub account to my Georgia Tech email ID.
2. Requested and received an official document from  Georgia Tech's Office of Registrar showcasing my status as an online MS student with College of Engineering. This document is signed by the registrar and has some additional security threads in it.
3. Made sure the name on my profile matches that in the official documentation.

What I cannot do:
1. Be physically present in Atlanta.
2. Convince GitHub's algorithms that this official document  - printed on SCRIP-SAFE(r) security paper and kindly sent by the Office of Registrar in Atlanta to all the way here in Singapore - is somehow a valid piece of document.
3. Get any form of official human response to my previous query dated 12 Aug 2023. 

I still keep getting this error message: 
<img width="732" alt="image" src="https://github.com/community/community/assets/4099802/4f78249d-0045-4237-a890-6dbd8c70e822">

I'm at my wit's end as to what else I can do. The term has started and I would like to use private repos to track my term projects. 

My earnest request to GitHub is to publicly state that they can't support enrolled students in Georgia Tech with their education pack. Would save us all a lot of grief and headache.

**Comment by arafajs** [comment]
1. Make sure that your GitHub profile name matches the name on your official document exactly. Even a small difference, such as a middle initial, could cause the verification to fail.
2. Try uploading a different document. If you have a transcript or a class schedule, that could also be sufficient to verify your student status.
3. Contact GitHub Education support again. Explain that you are a remote student and that you have provided a valid document proving your student status. Be sure to include the tracking number for your previous ticket.

**Reply by metamutator** [reply]
I have updated by Github profile exactly. I have submitted _three_ separate sets of documents, my Georgia Tech Buzzcard, a PDF of my class schedule, and a formal document from the Office of Registrar stating that I'm a student (as mentioned in this post).

Could you please suggest how I can contact Github Education support? I was originally trying to do so (per the screenshot), but the screenshot is simply wrong: all I can get is a virtual assistant at support.github.com, and a suggestion to post here in the community. I have not been able to find a way to raise an actual support ticket in this regard. 
![Screenshot 2023-08-30 at 11 47 36 AM](https://github.com/community/community/assets/4099802/af48b03c-f6fb-4392-97c6-02a73ecb206a)

Following this instruction, I had previously posted here: https://github.com/orgs/community/discussions/63551  I had received zero updates from Github or anyone. Posted here again because I was able to update the documentation.

**Reply by sascharo** [reply]
Has the issue been resolved for you?

**Reply by metamutator** [reply]
Nope. More like, I've given up.

**Reply by sascharo** [reply]
> Nope. More like, I've given up.

Mine got resolved a few days ago finally.
